media/transcoder/create_job_with_concatenated_inputs.py

```diff
@@ -0,0 +1,202 @@
+#!/usr/bin/env python
+
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Google Cloud Transcoder sample for creating a job based on concatenating two input videos.
+
+Example usage:
+    python create_job_with_concatenated_inputs.py --project_id <project-id> --location <location> \
+      --input1_uri <uri> --start_time_input1 <sec> --end_time_input1 <sec> \
+      --input2_uri <uri> --start_time_input2 <sec> --end_time_input2 <sec> \
+      --output_uri <uri>
+"""
+
+# [START transcoder_create_job_with_concatenated_inputs]
+
+import argparse
+
+from google.cloud.video import transcoder_v1
+from google.cloud.video.transcoder_v1.services.transcoder_service import (
+    TranscoderServiceClient,
+)
+from google.protobuf import duration_pb2 as duration
+
+
+def create_job_with_concatenated_inputs(
+    project_id,
+    location,
+    input1_uri,
+    start_time_input1,
+    end_time_input1,
+    input2_uri,
+    start_time_input2,
+    end_time_input2,
+    output_uri,
+):
+    """Creates a job based on an ad-hoc job configuration that concatenates two input videos.
+
+    Args:
+        project_id: The GCP project ID.
+        location: The location to start the job in.
+        input1_uri: Uri of the first video in the Cloud Storage bucket.
+        start_time_input1: Start time, in fractional seconds ending in 's'
+          (e.g., '0s'), relative to the first input video timeline.
+        end_time_input1: End time, in fractional seconds ending in 's'
+          (e.g., '8.1s'), relative to the first input video timeline.
+        input2_uri: Uri of the second video in the Cloud Storage bucket.
+        start_time_input2: Start time, in fractional seconds ending in 's'
+          (e.g., '3.5s'), relative to the second input video timeline.
+        end_time_input2: End time, in fractional seconds ending in 's'
+          (e.g., '15s'), relative to the second input video timeline.
+        output_uri: Uri of the video output folder in the Cloud Storage bucket."""
```

Comment:
Please add types for these args:

```
project_id (str): The GCP project ID.
```

Reply:
I can do this, but every Python code sample for Transcoder uses string args exclusively.

Reply:
Done